hieradata/cluster/chango.yaml

```diff
@@ -0,0 +1,21 @@
+---
+clustershell::groupmembers:
+  chango: {group: "chango", member: "chango[01-03]"}
+cni::plugins::checksum: "f3a841324845ca6bf0d4091b4fc7f97e18a623172158b72fc3fdcdb9d42d2d37"
+cni::plugins::enable: ["macvlan"]
+cni::plugins::version: "1.2.0"
```

Comment:
As there are no users of multus on this cluster, all of the `cni::*` keys should be removed.

Reply:
Removed from configuraiton.